### PR TITLE
feat(OCPADVISOR-123): filter configs filter slice and constants

### PR DESCRIPTION
**Pull request by Fewwy**
https://issues.redhat.com/browse/OCPADVISOR-123

### CODE REVIEW

File: src/Services/Filters.js [hunk 1]
```diff
@@ -44,11 +44,21 @@ export const CLUSTER_RULES_INITIAL_STATE = {
   text: '',
 };
 
+export const WORKLOADS_LIST_INITIAL_STATE = {
+  limit: 50,
+  offset: 0,
+  sortIndex: CLUSTERS_TABLE_CELL_LAST_SEEN,
```
Comment (gkarat) [inline src/Services/Filters.js:50]: Should we use the constant for the workloads table instead?

**Review by gkarat — APPROVED** [review]
looks good to me! left a few comments

File: src/Services/Filters.js (hunk 1 above)
Comment (gkarat) [reply src/Services/Filters.js:50]: Why renaming? I think creating a new one is the right way since these are two separate tables.

**Review by AsToNlele — APPROVED** [review]
LGTM, good job! :smile:

File: src/Services/Filters.js (hunk 1 above)
Comment (Fewwy) [reply src/Services/Filters.js:50]: Aaaaaaaaah! You mean for the next table! Yes, there will be a constant for it as well, not in this PR though :D

Comment (Fewwy) [reply src/Services/Filters.js:50]: I'm stupid...for some reason, I thought you were talking about the object itself...

**Review by AsToNlele — APPROVED** [review]
Good job! The filters look like in the mocks, should be straightforward to connect with the table. :smile:

**Pull request merged**